bin/rad-project

```diff
@@ -57,9 +57,20 @@
     (fetch! rsm)
     (def name (lookup :name (get-meta! rsm)))
     (def origin (lookup :id (first-rsm-of-type! rsm :rad-repo)))
-    (put-str! name)
-    (process! "git" ["clone" origin name] "")
-    (cd! name)
+    (match (process! "test" ["-d" name] "")
+      :ok (do
+            (put-str! (string-append
+                        "The directory \"" name "\" already exists. "
+                        "It needs to be removed to checkout the project in this directory."))
```

Comment:
Removing an existing dir is dangerous advice to give :) Maybe just "Refusing to continue."